### ISSUE TITLE
[$250] Web - Group Search - If user taps "Enter" on keyboard the previous users will be deleted

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#9009]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Access new.expensify.com
2. Sign into a valid account
3. Click on "+" icon and "New group"
4. Add 8 users, until the group is full (Message should appear)
5. Type in the search box a 9th user that has not been added yet
6. Tap "Enter" on the devices keyboard and repeat

## Expected Result:
The 9th user would be added to the group

## Actual Result:
All previously added users are deleted one by one. Only after every user is deleted, the searched user will receive a green tick and be added to the group 

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web

**Version Number:** 1.1.60.1

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** 

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/168399666-a3b12445-a32d-40f3-8144-62c871ebe564.mp4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause
 
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by roryabraham** [comment]
I recommend putting this on HOLD while I continue working on https://github.com/Expensify/App/pull/7702

**Comment by alex-mechler** [comment]
Still holding on the above pr melvin

**Comment by alex-mechler** [comment]
Still holding on the same pr

**Comment by alex-mechler** [comment]
This is off hold now, @kbecciv @mvtglobally @isagoico can we get this retested just to confirm its still an issue?

**Comment by stephanieelliott** [comment]
This is on prod, not just staging. Updated the issue body and repro steps for clarity.

**Comment by stephanieelliott** [comment]
Posted to Upwork: https://www.upwork.com/jobs/~0108fed7712b120377

**Comment by rushatgabhane** [comment]
Hurray! this issue isn't reproducible after #7702 

Here's how it behaves on staging.

https://user-images.githubusercontent.com/29673073/176569443-20373bcb-ffda-4eb6-82c3-3155cddcbede.mov

@aldo-expensify let's close this issue

**Comment by stephanieelliott** [comment]
Yay! Closing.
